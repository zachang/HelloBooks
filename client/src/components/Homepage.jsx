import React from 'react';
import HomepageHeader from './common/HomepageHeader';
import HomepageStatement from './HomepageStatement';

export default class Homepage extends React.Component {
  constructor(props){
    super(props);
    this.state = {
      showLoginText: false
    };
  }

  render() {
    return (
      <div className='row main-row main-div'>

        <HomepageHeader
          btnText={this.state.showLoginText}
        />

        <div className='section no-pad-bot' id='index-banner'>
          <div className='container textContent'>
            <div className='row'>

              <HomepageStatement/>

            </div>
          </div>

        </div>
      </div>
    );
  }
}
