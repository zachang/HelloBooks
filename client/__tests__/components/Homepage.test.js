import React from 'react';
import toJson from 'enzyme-to-json';
import sinon from 'sinon';
import { configure, shallow, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';

import Homepage from '../../src/components/Homepage.jsx';


configure({ adapter: new Adapter() });

const shallowWrapper = shallow(<Homepage/>);
const tree = toJson(shallowWrapper);
describe('<Homepage/>', () => {
  it('renders <Homepage/> component', () => {
    expect(tree).toMatchSnapshot();
  });
});
