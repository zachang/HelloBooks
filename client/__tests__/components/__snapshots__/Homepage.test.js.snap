// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Homepage/> renders <Homepage/> component 1`] = `
<div
  className="row main-row main-div"
>
  <HomepageHeader
    btnText={false}
  />
  <div
    className="section no-pad-bot"
    id="index-banner"
  >
    <div
      className="container textContent"
    >
      <div
        className="row"
      >
        <HomepageStatement />
      </div>
    </div>
  </div>
</div>
`;
